microsoft-edge/devtools-guide-chromium/whats-new/2021/09/devtools.md

```diff
@@ -0,0 +1,125 @@
+---
+description: Easily search the web for console error message strings, DevTools extension for Visual Studio Code now has the newest tools and theme support, breakpoint icons are now displayed when using Visual Studio Code themes, and you can navigate to the More Tools button with the keyboard.
+title: What's new in DevTools (Microsoft Edge 94)
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.date: 09/03/2021
+ms.topic: article
+ms.prod: microsoft-edge
+keywords: microsoft edge, web development, f12 tools, devtools
+---
+# What's New In DevTools (Microsoft Edge 94)
+
+[!INCLUDE [note about What's New announcements from the Microsoft Edge DevTools team](../../includes/edge-whats-new-note.md)]
+
+
+## Look up error messages on the web from the Console
+
+<!-- Title: Quickly debug console errors with our new integrated search feature -->
+<!-- Subtitle: Now you can quickly search for console errors directly from the Console. -->
+
+Search the web for your **Console** error strings, from within DevTools.  In the **Console**, an error message now has a **Search for this message on the Web** button, shown as a magnifying glass.  When you select the **Search for this message on the Web** button, a new tab opens in the browser and shows search results for the error string.
+
+<!-- todo: approve and merge PR 1430, update this branch from master, uncomment both link lines: -->
+<!-- For more information, navigate to [Looking up error messages on the web from the Console][LookupErrorsWebFromConsole]. -->
+
+:::image type="complex" source="../../media/2021/09/search-console-icon.msft.png" alt-text="The 'Search for this message on the Web' button on an error message in the **Console**" lightbox="../../media/2021/09/search-console-icon.msft.png":::
+   The **Search for this message on the Web** button on an error message in the **Console**
+:::image-end:::
+
+
+## DevTools extension for Visual Studio Code includes the latest tools, theme support, and helpful links
+
+<!-- Title: Edge DevTools for VS Code now supports themes and uses the most recent codebase -->
+<!-- Subtitle: The Edge DevTools extension for VS Code now uses the same version of the Developer Tools as your Microsoft Edge browser. We also added ways to learn more and for you to tell us what we could do better from within Visual Studio Code. -->
+
+In this release of the Microsoft Edge DevTools extension for Visual Studio Code, we released the following updates or new features.
+*  Shares the same codebase that's used for the browser-based DevTools. 
+*  Supports themes that ship with Visual Studio Code.
+*  The icon on the sidebar provides links to documentation and access to file bugs or request new features.  
+*  Includes a **Close** button.
```

Comment:
@codepo8 - where is the Close button?  I'm looking at the captures in the public docs and the Edge DevTools tab already has a Close button.